### DISCUSSION TITLE
主题右侧的滑动工具在哪里添加的？

### DISCUSSION THREAD

**Discussion by dlbmoe** [Tokinx/Wing#99] [问答] [answered]
![image](https://user-images.githubusercontent.com/34018267/219354874-bf40626b-0798-4c0a-aa15-4194a7c22383.png)
大家好，如上图，我在主题里面没有找到对应的设置。

**Comment by noteMay** [comment]
> ![image](https://user-images.githubusercontent.com/34018267/219354874-bf40626b-0798-4c0a-aa15-4194a7c22383.png) 大家好，如上图，我在主题里面没有找到对应的设置。

自带的吧！

**Reply by dlbmoe** [reply]
![image](https://user-images.githubusercontent.com/34018267/219535678-871b3a9b-dcf8-458f-9601-989f3caeef93.png)
自带的是在这里，跟上图的不一样

**Reply by Tokinx** [reply]
后台没收到更新推送吗？
收不到推送的话可以尝试从github下载一份更新一下主题

**Comment by Tokinx** [comment, marked as answer]
更新到最新版即可

**Reply by dlbmoe** [reply]
好的，谢谢
